### PR TITLE
Fastlane Setup & Google Play Alpha Deployment

### PR DESCRIPTION
**Pull request by PSchmiedmayer**
# Fastlane Setup & Google Play Alpha Deployment

## :recycle: Current situation & Problem
- The GitHub Actions currently only create an apk that is not downloadable from the Google Play store 

## :gear: Release Notes 
- Adds an initial fastlane setup
- Adds GitHub Action to publish to the play store
- Adds content metadata about the ENGAGE-HF app

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by PSchmiedmayer** [comment]
@eldcn @Basler182 @pauljohanneskraft @vishnuravi This PR adds a setup for fastlane as discussed in our last meeting on Tuesday. I can verify that the upload to Google Play works, unfortunately we currently don't have a fully verified organization account yet (in the works) and my personal is missing an Android device (is in the office to add this) we we can only really resume testing this on Monday but should have something solid by Tuesday.

Let me know if you already have some feedback based on the current diff 👍

File: fastlane/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+fastlane documentation
+----
+
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
```
Comment (Basler182) [inline fastlane/README.md:6]: did this possibly happen when copying the docu from iOS? or do you really need xcode?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -12,28 +20,39 @@ jobs:
...
-        uses: actions/upload-artifact@v2
-        with:
-          name: app
-          path: app/build/outputs/apk/release/app-release-unsigned.apk
-
+        uses: actions/checkout@v4
+      - name: Bundle Install
+        run: |
+          gem install bundler
+          bundle install
```
Comment (Basler182) [inline .github/workflows/main.yml:34]: minor: so far i think all steps had only one single command that was executed. we could think about splitting this line into two steps: instann bundler and bundle install

**Review by Basler182 — APPROVED** [review]
Many thanks for your PR. In general, it looks good. I have added two small comments.

File: fastlane/README.md (hunk 1 above)
Comment (PSchmiedmayer) [reply fastlane/README.md:6]: Funny enough no; this README is automatically generated when a fastlane command is run locally. We could remove the README from the directory. Fastlane is primarily used for Apple platforms but has a decent Android support as well.

File: .github/workflows/main.yml (hunk 2 above)
Comment (PSchmiedmayer) [reply .github/workflows/main.yml:34]: I think its fine if we keep corresponding steps in a single run execution, we do a similar thing in our reusable workflow and try to group them by "step", e.g., installing dependencies.

**Pull request merged**